NLog.Targets.OpenTelemetryProtocol/OtlpTarget.cs

```diff
@@ -68,14 +74,14 @@ protected override void InitializeTarget()
 #if TEST
             LogRecords = new List<LogRecord>();
 #endif
-            var endpoint = RenderLogEvent(Endpoint, LogEventInfo.CreateNullEvent());
-            var useHttp = RenderLogEvent(UseHttp, LogEventInfo.CreateNullEvent());
-            var headers = RenderLogEvent(Headers, LogEventInfo.CreateNullEvent());
+
             var maxQueueSize = RenderLogEvent(MaxQueueSize, LogEventInfo.CreateNullEvent(), 2048);
             var maxExportBatchSize = RenderLogEvent(MaxExportBatchSize, LogEventInfo.CreateNullEvent(), 512);
             var scheduledDelayMilliseconds = RenderLogEvent(ScheduledDelayMilliseconds, LogEventInfo.CreateNullEvent(), 5000);
 
-            _processor = CreateProcessor(endpoint, useHttp, headers, maxQueueSize, maxExportBatchSize, scheduledDelayMilliseconds);
+            var options = UseInjectedOptions ? GetInjectedOtlpExporterOptions() : GetOtlpExporterOptions();
```

Comment:
Maybe make it like this:
```c#
var options = ResolveOtlpExporterOptionsFromName() ?? GetOtlpExporterOptions();
```
Where `ResolveOtlpExporterOptionsFromName` returns `null`, when `ResolveOptionsFromName` renders into null or empty-string.